.flowconfig

```diff
@@ -1,5 +1,6 @@
 [ignore]
 .*/node_modules/config-chain/test/broken.json
+.*/src/components/__tests__/**/*
```

Comment:
why not to use flow in tests?

Reply:
Because some test cases can be run with incorrect data, but we can use flow-ignore 🤔
I will fix it.